### ISSUE TITLE
Cannot install 0.3.0

### ISSUE THREAD

**Issue by SirPoot-** [alexweissman/UserFrosting#340]
Hello Alex,

I am attempting to install version 0.3.0. I downloaded the build as of today. I am running locally on a WAMP server. I extracted the content of public folder into the public www folder and userfrosting under the root folder. I configured the DB settings. When .htaccess (from the download) is in the www folder, I am receiving error 500 "Internal Server Error". After removing the .htaccess, I a 404 error "Not Found. The requested URL /install/ was not found on this server." I am unable to bring up the setup the admin access page. 

I am really excited for the SB Admin 2 integration.

Any recommendations?

- Devin

**Comment by alexweissman** [comment]
You'll definitely need the `.htaccess` file in order for URL routing to work properly.  When you have it in place, did you get any error messages in your PHP error log?

**Comment by SirPoot-** [comment]
Alex,

I did not have any log entries in my php error log; however, I did have entries in my Apache error log.  

[Tue Jun 09 07:15:55.989706 2015] [core:alert] [pid 3280:tid 820] [client 192.168.1.165:60175] C:/wamp/www/.htaccess: Invalid command 'RewriteEngine', perhaps misspelled or defined by a module not included in the server configuration

I enabled the RewriteEngine module and I was able to install it.

Thank you for the quick response.


- Devin
